### PR TITLE
Update MPI migrate function

### PR DESCRIPTION
**Pull request by csegarragonz**
Changes:
* Make the number of loops an input parameter and not a constant.
* Measure the time spent migrating for migrated and non-migrated ranks.

The latter measure is informative, so it does not really matter that we measure within WASM.

### CODE REVIEW

File: func/mpi/migrate.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include "faasm/time.h"
```
Comment (Shillaker) [inline func/mpi/migrate.cpp:1]: Can put this import with the others below in `<>`

File: func/mpi/migrate.cpp [hunk 2]
```diff
@@ -62,6 +63,14 @@ void doBenchmark(int nLoops)
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &worldSize);
 
+    // Measure time spent migrating
+    // timespec timeStart{}, timeEnd{};
```
Comment (Shillaker) [inline func/mpi/migrate.cpp:67]: Does this line need to be here commented out? Can we delete?

**Review by Shillaker — CHANGES_REQUESTED** [review]
Couple of small things.

File: func/mpi/migrate.cpp (hunk 2 above)
Comment (csegarragonz) [reply func/mpi/migrate.cpp]: I forgot to remove this one :/

Thanks

**Review by Shillaker — APPROVED** [review]

**Pull request merged**